packages/assistify-help-request/client/views/creationDialog/AssistifyCreateRequest.js

```diff
@@ -2,20 +2,11 @@
 import {RocketChat} from 'meteor/rocketchat:lib';
 import {FlowRouter} from 'meteor/kadira:flow-router';
 import {ReactiveVar} from 'meteor/reactive-var';
-import toastr from 'toastr';
-
-const validateRequestName = (name) => {
```

Comment:
moved this to the template instance ('onCreated'), see http://blazejs.org/guide/reusable-components.html#Attach-functions-to-the-instance